packages/nft/src/NFTHandler.ts

```diff
@@ -117,4 +120,68 @@ export class NFTHandler {
       tokenIds.map(tokenId => this.get(nftManager, storage, tokenId, nftFormat))
     )
   }
+
+  /**
+   * It takes a tokenId, and returns a transaction object
+   * @param {NFT721Manager} nftManager - NFT721Manager - The NFT721Manager instance that will be used
+   * to set the NFT tokenURI.
+   * @param {StorageType} storage - StorageType - The storage to save the NFT metadata.
+   * @param {NFT_METADATA_FORMAT} nftFormat - The format of the NFT metadata.
+   * @param {NFTData} nftData - The data of the NFT you want to set.
+   * @returns The transaction object
+   */
+  static async change(
+    nftManager: NFT721Manager,
+    storage: StorageType,
+    nftFormat: NFT_METADATA_FORMAT,
+    nftData: NFTData
+  ): Promise<interfaces.TransactionResponseExtended | undefined> {
```

Comment:
I think this should be `ExecuteResponse` instead of `TransactionResponseExtended | undefined`. That's what the `execute` returns